### PR TITLE
Add main page

### LINKED ISSUE #114: Pages route should not show a list of pages
Should instead contain some information about the student division.

Compare [https://chalmers.it/sektionen/](https://chalmers.it/sektionen/) and [https://beta.chalmers.it/pages](https://beta.chalmers.it/pages)

### PR DESCRIPTION
**Pull request by Kerp1**
Add a "main page" that is displayed when navigating to /pages. Thanks @Kalior 

Remember to run rake db:seed :dancing_men: 

Fixes #114

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -63,6 +63,7 @@
                     <li><%= link_to t('.edit_profile'), edit_user_path %></li>
                     <% if current_user.admin? %>
                       <li><%= link_to t('.admin'), '#admin_path', target: '_blank' %></li>
+                      <li><%= link_to t('.edit_frontpage'), edit_frontpage_path %></li>
```
Comment (Vilddjur) [inline app/views/layouts/_header.html.erb]: This has some weird consequences with the styling

**Comment by Vilddjur** [comment]
Not sure if this is because the routeing fix is not in the PR but the links look weird when they are in your default(?) locale. I.e swedish link: http://vilddjur.chalmers.it:3000/sv/student-division
vs english link http://vilddjur.chalmers.it:3000//student-division
seems to only be on links that refer to the page on the site, http://vilddjur.chalmers.it:3000/student-division works but if i navigate through the sidebar i get the awkward links

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (Vilddjur) [inline app/views/layouts/_header.html.erb]: This issue seems to exist on other branches aswell.

**Comment by lindskogen** [comment]
@Vilddjur probably because https://github.com/cthit/chalmersit-rails/blob/develop/app/helpers/pages_helper.rb is a bit unstable

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (Kalior) [inline app/views/layouts/_header.html.erb]: @Kerp1 had a theory that this was because of his recent PR that centered the contents on that row.

**Comment by Kalior** [comment]
Yeah, that issue exists live as well: https://beta.chalmers.it//dokument when navigating through the sidebar

**Pull request merged**